lack/src/test/java/io/datanerds/lack/LackTest.java

```diff
@@ -3,15 +3,14 @@
 import io.datanerds.lack.cassandra.LackConfig;
 import org.cassandraunit.CassandraCQLUnit;
 import org.cassandraunit.dataset.cql.ClassPathCQLDataSet;
-import org.cassandraunit.utils.EmbeddedCassandraServerHelper;
 import org.junit.*;
 import org.junit.rules.ExpectedException;
 
 import java.util.UUID;
 
 public class LackTest {
 
-    private static LackConfig config = new LackConfig(null, null, new String[]{"127.0.0.1"}, 9142, "lack", 1);
+    private static LackConfig config = new LackConfig(null, null, new String[]{"127.0.0.1"}, 9142, "lack", 3);
```

Comment:
Accuracy in C* for ttl is seconds. So in order to test the renew functionality I needed some more time :-) (e.g. renewing after 500 milliseconds won't renew the lock)